sql/create.sql

```diff
@@ -1,5 +1,7 @@
+DROP DATABASE IF EXISTS flightdb2;
```

Comment:
Destructive!

Arguably all the individual `DROP TABLE IF EXISTS` are now redundant... Or your mariadb/mysql is buggy AF if it manages to drop tables at those points

Reply:
They were redundant before this change too since `create database` would error out if one already existed. But if it didn't, then it'd be destructive since it would drop all the tables.

Perhaps the other changes shouldn't have been merged back into `create.sql` and the scripts should have been run as database migrations...